console/command/Test.php

```diff
@@ -59,6 +59,13 @@ class Test extends \lithium\console\Command {
 	public $verbose = false;
 
 	/**
+	 * Shows text output with color or without
+	 *
+	 * @var boolean
+	 */
+	public $no_color = false;
```

Comment:
Reverted this, because it is necessary for the 'help' command. It doesn't seem to take into account the inherited properties.

I am now following the setup for the rest of the properties (plain is defined in both Command and Test)